frontend/src/components/project/utils/getTariffs.ts

```diff
@@ -17,10 +17,10 @@ export function getTariffs(projectClient: ProjectClientModel): ProjectClientTari
   };
 }
 
-export const compareTariffs = (t1: ProjectClientTariff, t2: ProjectClientTariff, seperateHourlyDaily: boolean = true): number => {
-  if(!seperateHourlyDaily)
-    return (t2.rateType === 'hourly' ? t2.tariff * DefaultHoursInDay : t2.tariff) -
-      (t1.rateType === 'hourly' ? t1.tariff * DefaultHoursInDay : t1.tariff)
+export const compareTariffs = (t1: ProjectClientTariff, t2: ProjectClientTariff, separateHourlyDaily: boolean = true): number => {
+  if(!separateHourlyDaily)
+    return (t1.rateType === 'hourly' ? DefaultHoursInDay : 1) * t1.tariff -
+      (t2.rateType === 'hourly' ? DefaultHoursInDay : 1) * t2.tariff
```

Comment:
Check je mijn comment in vorige PR?

```
const t1Tariff = t1.tariff * (t1.rateType === 'hourly' ? DefaultHoursInDay : 1);
const t2Tariff = t2.tariff * (t2.rateType === 'hourly' ? DefaultHoursInDay : 1);
return t1Tariff - t2Tariff;
```

Ik denk dat je het altijd op die manier kan sorteren en dat je dan die EditProjectRateTypeSortOrder niet meer nodig hebt en ook de separateHourlyDaily: boolean parameter niet meer?